### ISSUE TITLE
[Update] Better Organize "Getting Started" Sections

### ISSUE THREAD

**Issue by AdultSun** [stashapp/StashDB-Docs#29]
While thinking about the best way to organize the request detailed in issue #28, I'm wondering if we could better organize our "Getting Started" sections. This is essentially a braindump of an idea I had 2 minutes ago so bear with me.

So in Stash-Docs, we have a heading in the sidebar to collect [Beginner Guides](https://docs.stashapp.cc/docs/Beginner-Guides/) including our often recommended [Guide to Scraping](https://docs.stashapp.cc/docs/Beginner-Guides/Guide-To-Scraping/). Organizing it this way allows us to point at a single location for anyone looking to get started with Stash for the first time, explaining any important concepts/features that may not be immediately obvious within the application's UI. A lot of our "Getting Started" sections for StashDB function the same way, but the formatting there mirrors the "user manual" approach to each individual section of the actual guidelines that need "official" approval. This makes some pages more cluttered than they need to be and fragments some others. I can see it confusing the overall formatting of the site to new contributors.

I'm thinking a similar setup to the "Beginner Guides" in Stash-Docs may be better. We can collect all the "Getting Started" sections from the various headings to de-clutter those sections a bit, clearly separate "guides" from "guidelines" (the rulebook sections essentially), and allow for more flexibility in their formatting (i.e., more pages patterned after the step-by-step approach used in the [Guide to Scraping](https://docs.stashapp.cc/docs/Beginner-Guides/Guide-To-Scraping/)). We could have a "Guide to Performers/Scenes/Studios/Tags" made up of the various "Getting Started" sections currently under their respective headings. We could also add new stand-alone pages for "Guide to Joining", "Guide to Contributing", "Guide to JAV", and whatever else we think could be useful.

Moving all these sections around would also likely break a lot of internal links, so we'd have to be thorough with a project like this. For anyone reading, let me know if you think this would be a more intuitive way to organize these topics. Either way, I've never been totally happy with the way everything's organized since a lot of those decisions were made due to the limits of the original GitHub wiki it was hosted on. We've got a lot more flexibility with the new Jekyll-based system and I don't think we've taken full advantage of the differences yet, so suggestions on that front are always welcome.

**Comment by DogmaDragon** [comment]
I agree with getting rid of "Getting Started" they feel out of place and indirectly imply that reading them is enough to understand the guidelines when it's actually not. 

I'm not sure how I feel about having guides for each metadata category. I think it would still duplicate information with guidelines. In my opinion, having "guides" integrated into the same sections as guidelines would feed more user-friendly. Just adding a heading with `best practices` where appropriate or as [callouts](https://just-the-docs.github.io/just-the-docs/docs/ui-components/callouts/).
